src/utils/findGitmojiCommand.ts

```diff
@@ -0,0 +1,86 @@
+// @flow
+import { CommitModes } from '@constants/commit'; // Assuming COMMIT_MODES is exported as CommitModes
+import FLAGS from '@constants/flags';
+
+interface Options {
+  [key: string]: any;
+}
+
+interface Cli {
+  flags: { [key: string]: any };
+  input: string[];
+  showHelp(): void;
+}
+
+const isSupportedCommand = (command: string | undefined, options: Options): boolean => {
+  return command !== undefined && Object.keys(options).includes(command);
+};
+
+type CommandType = 'flag' | 'command';
```

Comment:
PP: Use `enum`'s where they make sense.

```suggestion
enum CommandType {
  Flag =  'flag',
  Command = 'command',
}
```